File: Morning-snacks/lowest_common_multiple.py
first_number = int(input("Enter a number: "))
second_number = int(input("Enter a number: "))

lcm = 0

if first_number >= second_number:
	lcm = first_number
else:
	lcm = second_number

while True:
	if lcm % first_number == 0 and lcm % second_number == 0:
		break
	lcm += 1

print("The LCM is ", lcm)

File: Morning-snacks/LowestCommonMultiple.java
 import java.util.Scanner;

public class LowestCommonMultiple{

	public static void main(String... args){

		Scanner input = new Scanner(System.in);

		System.out.print("Enter first number: ");
		int firstNumber = input.nextInt();

		System.out.print("Enter second number: ");
		int secondNumber = input.nextInt();

		int lcm = 0;

		if(firstNumber >= secondNumber){
			lcm = firstNumber;
		}else{
			lcm = secondNumber;
		}

		while (true) {

			if(lcm % firstNumber == 0 && lcm % secondNumber == 0){
				break;
			}
		
		lcm++;
		}

		System.out.println("The LCM is " + lcm);

	}
}